packages/ibm-products-styles/src/components/EditTearsheet/_index.scss

```diff
@@ -1,5 +1,5 @@
 //
-// Copyright IBM Corp. 2022, 2022
```

Comment:
Did this file get updated?

Reply:
this was probably leftover from something else. i'll fix it